articles/intro-to-katas.md

```diff
@@ -27,47 +28,59 @@ You can use the Katas for learning in your environment of choice:
+|[Deutsch–Jozsa algorithm](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ExploringDeutschJozsaAlgorithm)|This algorithm is famous for being one of the first examples of a quantum algorithm that is exponentially faster than any deterministic classical algorithm.|
+|[Exploring high-level properties of Grover's search algorithm](https://github.com/microsoft/QuantumKatas/tree/master/tutorials/ExploringGroversAlgorithm)|A high-level introduction to one of the most famous algorithms in quantum computing. It solves the problem of finding an input to a black box (oracle) that produces a particular output. |
+|[Implementing Grover's search algorithm](https://github.com/microsoft/QuantumKatas/tree/master/GroversAlgorithm)|This kata dives deeper into Grover's search algorithm, and covers writing oracles, performing steps of the algorithm, and finally putting it all together.|
+|[SAT problems](https://github.com/microsoft/QuantumKatas/tree/master/SolveSATWithGrover)|A series of exercises that uses Grover's algorithm to solve realistic problems, using [boolean satisfiability problems](https://en.wikipedia.org/wiki/Boolean_satisfiability_problem) (SAT) as an example.  |
+|[Graph coloring problems](https://github.com/microsoft/QuantumKatas/tree/master/GraphColoring)| This kata further explores Grover's algorithm, this time to solve [constraint satisfaction problems](https://en.wikipedia.org/wiki/Constraint_satisfaction_problem), using a graft coloring problem as an example. |
 
 ### Protocols and libraries
 
-* [BB84 protocol for quantum key distribution](https://github.com/microsoft/QuantumKatas/tree/master/KeyDistribution_BB84)
-* Quantum error correction: [bit-flip error correcting code](https://github.com/microsoft/QuantumKatas/tree/master/QEC_BitFlipCode)
-* [Phase estimation](https://github.com/microsoft/QuantumKatas/blob/master/PhaseEstimation)
-* Quantum arithmetic: [building ripple-carry adders](https://github.com/microsoft/QuantumKatas/blob/master/RippleCarryAdder)
+| Kata | Description |
+|:-----|-------------|
+|[BB84 protocol for quantum key distribution](https://github.com/microsoft/QuantumKatas/tree/master/KeyDistribution_BB84)|Learn about and implement a quantum key distribution protocol, [BB84](https://en.wikipedia.org/wiki/BB84), using qubits to exchange cryptographic keys. |
+|[Bit-flip error correcting code](https://github.com/microsoft/QuantumKatas/tree/master/QEC_BitFlipCode)|Explore quantum error correction with the simplest of the quantum error-correction (QEC) codes - the three-qubit bit-flip code.|
+|[Phase estimation](https://github.com/microsoft/QuantumKatas/blob/master/PhaseEstimation)|Phase estimation algorithms are some of the most fundamental building blocks of quantum computing. Learn about phase estimation with these exercises that cover quantum phase estimation, iterative phase estimation, and how to prepare and run phase estimation routines in Q#.|
```

Comment:
I wouldn't emphasize iterative PE here, since it's not really well-covered...
```suggestion
|[Phase estimation](https://github.com/microsoft/QuantumKatas/blob/master/PhaseEstimation)|Phase estimation algorithms are some of the most fundamental building blocks of quantum computing. Learn about phase estimation with these exercises that cover quantum phase estimation and how to prepare and run phase estimation routines in Q#.|
```